lib/iris/coords.py

```diff
@@ -1600,9 +1585,6 @@ def _sanitise_array(self, src, ndmin):
         # Ensure the array has enough dimensions.
         # NB. Returns the *same object* if result.ndim >= ndmin
         result = np.array(result, ndmin=ndmin, copy=False)
-        # We don't need to copy the data, but we do need to have our
-        # own view so we can control the shape, etc.
-        result = result.view()
```

Comment:
Why do we no longer want to make a view of `result`? Is this because we've removed `LazyArray` or is it more in depth than that?